### ISSUE TITLE
Write Error when initializing auto-properties

### ISSUE THREAD

**Issue by cklenk** [influxdata/influxdb-client-csharp#586]
__Steps to reproduce:__

1. Set up measurement table with auto props with inline default values
2. Write query by POCO
3. Error

__Expected behavior:__
Able to write to measurement

__Actual behavior:__
Got error says unable to determine measurement. Does it have a InfluxDB.Client.Core.Measurement or IsMeasurement InfluxDB.Client.Core.Column attribute?

__Specifications:__
 - Client Version: 4.13.0
 - InfluxDB Version: 1.7
 - Platform: Linux

**Comment by bednar** [comment]
Hi @cklenk,

Thank you for using our client. Could you share some code that reproduces the exception you're encountering?

Regards

**Comment by cklenk** [comment]
> Hi @cklenk,
> 
> Thank you for using our client. Could you share some code that reproduces the exception you're encountering?
> 
> Regards

Yes, sure thing! Here is some code snippet. I think it should give you the gist of the problem. If I remove the default vars on the AlgoData class, the write works fine. Otherwise, I get that error. It's not a big deal, I have a method that will init the vars for me, but I do like to do it in 1 line on the class if possible.

`[Measurement("AlgoTable")]
class AlgoData
{
	[Column("DeviceId", IsTag = true)] public string DeviceId { get; set; }
	[Column("input")] public double InputTemp { get; set; }
	[Column("output")] public double OutputTemp { get; set; }
	[Column("peakValue")] public int PeakValue { get; set; } = 0;
	[Column("runningState")] public int RunningState { get; set; } = 2;
	[Column("signal")] public int Signal { get; set; } = 0;
	[Column(IsTimestamp = true)] public DateTime TimeStamp { get; set; }
}
async void InsertAlgoData(List<AlgoData> algoResults)
{
	using (InfluxDBClient client = GetClient(100))
	{
		try
		{
			// Write the measurement to database
			await client.GetWriteApiAsync().WriteMeasurementsAsync(algoResults, WritePrecision.Ns);
		}
		catch (Exception ex)
		{
			Exceptions.LogException(ex);
		}
	}
}
InfluxDBClient GetClient(double timespanSeconds = 10)
{
	var options = new InfluxDBClientOptions(url: "localhost")
	{
		Timeout = TimeSpan.FromSeconds(timespanSeconds),
		LogLevel = LogLevel.Basic,
		Org = "123",
		Bucket = "XX"
	};
	return new InfluxDBClient(options);
}`
